app/controllers/forms_controller.rb

```diff
@@ -91,24 +91,15 @@ def form_navigation
   end
 
   def member_params
-    filter_params(form_class.member_attributes)
+    filtered_params(form_class.member_attributes)
   end
 
   def application_params
-    filter_params(form_class.application_attributes)
+    filtered_params(form_class.application_attributes)
   end
 
   def navigator_params
-    filter_params(form_class.navigator_attributes)
-  end
-
-  def filter_params(attributes)
```

Comment:
Note: we don't need this anymore, since we no longer send birthdays as multi-part forms